Sofascore/Sofascore/EventView.swift

```diff
@@ -0,0 +1,211 @@
+    let awayTeamScore: UILabel = .init()
+    let homeTeamLogo: UIImageView = .init()
+    let awayTeamLogo: UIImageView = .init()
+    let timeLabel: UILabel = .init()
+    let statusLabel: UILabel = .init()
+    let divider: UIView = .init()
+    
+    override func addViews() {
+        addSubview(eventHorizontalStack)
+        eventHorizontalStack.axis = .horizontal
+        eventHorizontalStack.spacing = 16
+        
+        eventHorizontalStack.addArrangedSubview(matchStatusVerticalStack)
+        matchStatusVerticalStack.axis = .vertical
+        matchStatusVerticalStack.distribution = .fillEqually
+        matchStatusVerticalStack.addArrangedSubview(timeLabel)
+        matchStatusVerticalStack.addArrangedSubview(statusLabel)
+        timeLabel.textAlignment = .center
+        statusLabel.textAlignment = .center
+        
+        eventHorizontalStack.addArrangedSubview(divider)
+        
+        eventHorizontalStack.addArrangedSubview(teamInfoVerticalStack)
+        teamInfoVerticalStack.axis = .vertical
+        teamInfoVerticalStack.distribution = .fillEqually
+        
+        teamInfoVerticalStack.addArrangedSubview(homeTeamHorizontalStack)
+        homeTeamHorizontalStack.axis = .horizontal
+        homeTeamHorizontalStack.spacing = 8
+        homeTeamHorizontalStack.addArrangedSubview(homeTeamLogo)
+        homeTeamHorizontalStack.addArrangedSubview(homeTeamName)
+        homeTeamName.numberOfLines = 1
+        
+        teamInfoVerticalStack.addArrangedSubview(awayTeamHorizontalStack)
+        awayTeamHorizontalStack.axis = .horizontal
+        awayTeamHorizontalStack.spacing = 8
+        awayTeamHorizontalStack.addArrangedSubview(awayTeamLogo)
+        awayTeamHorizontalStack.addArrangedSubview(awayTeamName)
+        awayTeamName.numberOfLines = 1
+        
+        eventHorizontalStack.addArrangedSubview(scoreInfoVerticalStack)
+        scoreInfoVerticalStack.axis = .vertical
+        scoreInfoVerticalStack.distribution = .fillEqually
+        scoreInfoVerticalStack.addArrangedSubview(homeTeamScore)
+        homeTeamScore.numberOfLines = 1
+        scoreInfoVerticalStack.addArrangedSubview(awayTeamScore)
+        awayTeamScore.numberOfLines = 1
+        
+    }
+
+    override func styleViews() {
+        timeLabel.font = ApplicationsFonts.micro
+        timeLabel.textColor = ApplicationColors.secondaryColor
+        statusLabel.font = ApplicationsFonts.micro
+        homeTeamScore.font = ApplicationsFonts.body
+        awayTeamScore.font = ApplicationsFonts.body
+        homeTeamName.font = ApplicationsFonts.body
+        awayTeamName.font = ApplicationsFonts.body
+        divider.backgroundColor = ApplicationColors.dividerColor
+    }
+
+    override func setupConstraints() {
+        eventHorizontalStack.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.bottom.equalToSuperview()
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
+        }
```

Comment:
```suggestion
        eventHorizontalStack.snp.makeConstraints {
            $0.top.bottom.leading.trailing.equalToSuperview()
        }
```

Ako su ti isti constrainti, mozes ih chain-ati. Takodjer, ovdje se .top.bottom.leading.trailing moglo zamjeniti i s .edges